Core/src/main/java/com/plotsquared/core/player/PlotPlayer.java

```diff
@@ -841,6 +842,32 @@ public void sendTitle(
                 .title(titleComponent, subtitleComponent, times));
     }
 
+    /**
+     * This method is explicitly used for sending an ActionBar
+     * to a player when they enter a plot.
+     *
+     * This method will serve no purposes in other locations
+     * of the project.
+     *
+     * This takes in two captions as it will internally merge
+     * the captions together to send as a single component.
+     *
+     * @param header Header
+     * @param subHeader Subheader
+     * @param replacements Variable replacements
+     */
+    public void sendPlotEntryBar(
```

Comment:
I'd prefer to not have such a specific method in PlotPlayer. It's probably better to just do that stuff in the plot listener too and use getAudience there *or* just say we only use the header for the action bar and have a more general sendActionBar(Caption, Template...). That way, the content is also fully modifiable without the hardcoded `|`.

Not sure what the others think about it, but I'm in favor of second approach. That behavior just needs to be outlined in the settings comment then.

Reply:
This method is intended to be for titles only as of drafting it no implementation of actionbars but for titles has been considred.
If we want to add a more generalized method I'm not opposed to it.

Reply:
> This method is intended to be for titles only as of drafting it no implementation of actionbars but for titles has been considred.

Yes, that's too specific to end up in PlotPlayer in my opinion.The current implementation is also limited in the way it can be configured, that's why I brought up the second approach.

Reply:
I believe I attempted that original approach but after talking with NMF I could not (probably a misunderstanding on my end) find a reliable way to combine those two captions for the title and subtitle into a single spaced proper component to pass through cleanly.

Reply:
Yeah combining them always has some hardcoded magic and probably will never fulfill everyone's needs. That's why I'm suggesting to only use the title (badly worded before from my side) and ignore the subtitle for the action bar. If that's documented in the settings comment, I think it should be clear enough, we don't need a specialized method but only a `sendActionBar` one, and people can configure it the way they want.

Reply:
So you would only like to show the plot name that they are entering and drop the owner information when they are using actionbar?

Reply:
You can change the title entry in the messages to put it back in - it wouldn't be there by default tho. But I think that's still better than hardcoding a separator.

Reply:
Implemented!